_admin_guides/jira-integration.md

```diff
@@ -82,5 +82,109 @@ These IDs will be different for every workflow.  Note that if the workflow
 disallows the transition from the current stage to the stage given, this
 stage will fail.
 
+## Updating Jira Tickets on Deploy
 
+If you're using GitHub, you can also configure Spinnaker to update tickets
+based on the git commits that have been included a Deploy.
+
+To enable this feature, add the following to your `spinnaker-local.yml` file:
+
+```
```

Comment:
you can do 
```
```yaml
blah..... for yaml syntax heightliting
```

```

Reply:
I like it.  😄